src/components/ComposerAttachments.vue

```diff
@@ -172,21 +284,44 @@ export default {
 
 			try {
 				const paths = await picker.pick(t('mail', 'Choose a file to add as attachment'))
-				const fileSizes = await Promise.all(paths.map(getFileSize))
-				const newTotal = sum(fileSizes) + this.totalSizeOfUpload()
+				// maybe fiiled front with placeholder loader...?
+				const filesFromCloud = await Promise.all(paths.map(getFileData))
+
+				const sum = filesFromCloud.reduce((sum, item) => {
+					return sum + item.size
+				}, 0)
+
+				const newTotal = sum + this.totalSizeOfUpload()
+				const th = this
```

Comment:
```suggestion
```

Reply:
@st3iny I think everything can be checked. I tried to take everything into account and brought all the changes into 1 commit